packages/build/tests/helpers/normalize.js

```diff
@@ -132,15 +132,17 @@ const NORMALIZE_REGEXPS = [
   [/^(\s+"es2000"),/gm, '$1'],
   // Empty lines
   [/^ +$/gm, ''],
-  // Hexadecimal identifiers, like commit hash
-  [/[a-fA-F\d]{8,}/gm, 'HEXADECIMAL_ID'],
+  // Hexadecimal identifiers, like commit hash or UUID
+  [/[a-fA-F-\d]{8,}/gm, 'HEXADECIMAL_ID'],
   // HTTP errors are shown differently in Node 8
   [/ \.\.\.:443/g, ''],
   // List of available plugins from `plugins.json`.
   // That list changes all the time, so we need to remove it.
   [/(Available plugins)[^>]*/m, '$1\n\n'],
   // esbuild error messages
   [/(Could not resolve "[^"]+") \([^)]+\)/g, '$1'],
+  // Base64-encoded string
+  [/data:.*;base64,([a-zA-Z\d])+/g, 'BASE64_STRING'],
```

Comment:
```suggestion
  [/data:.*;base64,([a-zA-Z\d])+={0,2}/g, 'BASE64_STRING'],
```

Should we also include the possible two equal signs at then end?

Reply:
Oh, I hadn't realised you sent a suggestion in addition to the comment. I've pushed https://github.com/netlify/build/pull/4393/commits/b230eef65eb3e78f8fb66b1d0187ffa7793cc9f1 in the meantime.

I think the string can actually have more than two equal signs, so perhaps the more loose expression would work better?

Reply:
Not sure, according to SO only two. But as it is only for the tests, I guess it is fine as it is.

Reply:
Also your suggestion leaves the equal signs outside of the capture group, so they wouldn't get replaced. This means that two Base64 strings (one with equal signs, one without) wouldn't be normalised to the same thing. We could fix that, but as you say I'm not sure it's worth the trouble since it's for the test suite only.

Reply:
Ah okay, now I understand how this replacement logic works. :)